app/lib/frontend/templates/views/pkg/title_content.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../../../dom/dom.dart' as d;
+import '../../../static_files.dart';
+
+d.Node renderTitleContentNode({
+  required String package,
+  required String version,
+}) {
+  return d.fragment([
+    d.text('$package $version '),
+    d.span(
+      classes: ['pkg-page-title-copy'],
+      children: [
+        d.img(
+          classes: ['pkg-page-title-copy-icon'],
+          attributes: {
+            'data-copy-content': '$package: ^$version',
+            'data-ga-click-event': 'copy-package-version',
+          },
+          src: staticUrls.getAssetUrl('/static/img/content-copy-icon.svg'),
+          title: 'Copy "$package: ^$version" to clipboard',
+        ),
+        d.div(
+          classes: ['pkg-page-title-copy-feedback'],
+          children: [
+            d.span(classes: ['code'], text: '$package: ^$version'),
+            d.text('copied to clipboard'),
```

Comment:
Do we need a space in front? I guess we don't since the tests pass....
```suggestion
            d.text(' copied to clipboard'),
```

Reply:
That's a good point! I think the test pass, because we re-format the HTML and add a linebreak after the `<span>` in both cases.